tests/test_lab_evaluate.py

```diff
@@ -564,3 +564,31 @@ def test_vllm_args_null(cli_runner: CliRunner):
         ],
     )
     common.assert_tps(args, "4")
+
+
+@patch(
+    "instructlab.eval.mmlu.MMLUEvaluator.run",
+    return_value=(0.5, {"task1": {"score": 0.1}, "task2": {"score": 0.9}}),
+)
+def test_no_gpu_warning(run_mock, cli_runner: CliRunner):
+    fname = common.setup_gpus_config(section_path="serve", vllm_args=lambda: None)
```

Comment:
Not quite following this.  Doesn't the default config have null gpus configured?

Reply:
I've finally revisited this PR and was able to do this test without this